### PR TITLE
changed the class for each image

### PR DESCRIPTION
**Pull request by priyanshugarg1010**
Now it's working, I change the class name of each image to "eye".

### CODE REVIEW

File: proximity.css
Comment (Ravin972) [inline proximity.css:1]: This is a great improvement! The position of each eye class is now properly set. I like the use of ID selectors for each eye, it makes the code easier to read and understand. Good job!

Comment (ravin972) [inline proximity.css:1]: This is a great improvement! The position of each eye class is now properly set. I like the use of ID selectors for each eye, it makes the code easier to read and understand. Good job!

File: proximity.html [hunk 1]
```diff
@@ -31,19 +31,20 @@
 </head>
 
 <body>
-    <!-- Main content area -->
+    Main content area
```
Comment (ravin972) [inline proximity.html:34]: can we put this in comment section ?
I think we need to comment this line.

File: proximity.html
Comment (ravin972) [inline proximity.html:1]: Great!

File: proximity.js
Comment (ravin972) [inline proximity.js:1]: Great job , 
Thanks

**Review by ravin972 — APPROVED** [review]

**Pull request merged**